### PR TITLE
 A5: Pseudoclasses and Pseudoelements

### PR DESCRIPTION
**Pull request by Brandonz07**
completed

### CODE REVIEW

File: src/br/html/A5-Pseudoclasses-and-Pseudoelements/A5-Pseudoclasses-and-Pseudoelements.html [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        He leído y acepto los
+        <a href="#terminos" class="terms-link">Terminos y Condiciones</a>
+        <input type="checkbox" name="terms" required />
+        <input type="submit" value="Suscribirse" />
+      </form>
+
+      <div class="terminos" id="terminos">
+        <h2>Terminos y Condiciones</h2>
+        <ul class="terminos-lista">
+          <li class="li-one">
```
Comment (luisalejandrosalazar) [inline src/br/html/A5-Pseudoclasses-and-Pseudoelements/A5-Pseudoclasses-and-Pseudoelements.html:55]: Avoid create and use a class just for one element
Use the nth pseudo classes in order to select this component.

File: src/br/html/A5-Pseudoclasses-and-Pseudoelements/A5-Pseudoclasses-and-Pseudoelements.html [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+          name="phone"
+          placeholder="número de celular"
+        />
+        He leído y acepto los
+        <a href="#terminos" class="terms-link">Terminos y Condiciones</a>
+        <input type="checkbox" name="terms" required />
+        <input type="submit" value="Suscribirse" />
+      </form>
+
+      <div class="terminos" id="terminos">
```
Comment (luisalejandrosalazar) [inline src/br/html/A5-Pseudoclasses-and-Pseudoelements/A5-Pseudoclasses-and-Pseudoelements.html:52]: Avoid create and use a class just for one element
Please refactor it

File: src/br/html/A5-Pseudoclasses-and-Pseudoelements/styles/style.css [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+ul {
+  list-style: none;
+}
+
+.li-one {
+  color: #0747a6 !important;
+}
+
+.terminos-lista li:nth-child(odd) {
+  color: #5e6c84;
```
Comment (luisalejandrosalazar) [inline src/br/html/A5-Pseudoclasses-and-Pseudoelements/styles/style.css:58]: Stylize just the list icon.
Please refactor it

File: src/br/html/A5-Pseudoclasses-and-Pseudoelements/A5-Pseudoclasses-and-Pseudoelements.html (hunk 1 above)
Comment (Brandonz07) [reply src/br/html/A5-Pseudoclasses-and-Pseudoelements/A5-Pseudoclasses-and-Pseudoelements.html]: remove class and add pseudoclasses

File: src/br/html/A5-Pseudoclasses-and-Pseudoelements/styles/style.css [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+ul {
+  margin: 2%;
+}
+
+ul {
+  list-style: none;
+}
+
+.li-one {
+  color: #0747a6 !important;
```
[the remark this answers is not in the document]
Comment (Brandonz07) [reply src/br/html/A5-Pseudoclasses-and-Pseudoelements/styles/style.css]: remove class and !important

File: src/br/html/A5-Pseudoclasses-and-Pseudoelements/A5-Pseudoclasses-and-Pseudoelements.html (hunk 2 above)
Comment (Brandonz07) [reply src/br/html/A5-Pseudoclasses-and-Pseudoelements/A5-Pseudoclasses-and-Pseudoelements.html]: remove class and add pseudoclasses

File: src/br/html/A5-Pseudoclasses-and-Pseudoelements/styles/style.css (hunk 3 above)
Comment (Brandonz07) [reply src/br/html/A5-Pseudoclasses-and-Pseudoelements/styles/style.css:58]: just stylize the icons

**Pull request merged**